### PR TITLE
task_01.py

### PR DESCRIPTION
**Pull request by Tresanne**

### CODE REVIEW

**Comment by Tresanne** [comment]
Hi professor I need help. I keep getting an error that says "ACCENT NOT DEFINED". I do not understand why exactly. I run the file and everything for the Base = Seattle is fine. If i run it i get exactly what you have in the example but when i do the BASE = Manatee one, when it goes to the next line, it says ACCENT NOT DEFINED. I will copy and paste the error.
Traceback (most recent call last):
  File "/home/vagrant/Desktop/is210-week-04-synthesizing/task_01.py", line 10, in <module>
    if ACCENT == "Cermaic Glaze":
NameError: name 'ACCENT' is not defined
>>> 
I do not get it. What does Cermaic Glaze have to do with Manatee? It runs perfectly when I run the 'SEATTLE GRAY'.. o_o.

**Comment by Tresanne** [comment]
retest this please

**Comment by cheuschober** [comment]
@Tresanne, look into your indentation. These statements can be nested inside one another and, indeed, that's what's necessary to get this one to work. Your error is occurring because the logic chain you've built has some holes that are allowing the ACCENT variable to not be set.

**Comment by Tresanne** [comment]
Hi Professor. I am having issues with task 02 and task 01 and any feedback will help. For task 01 the only error is a space apparently but when i go into the editor and press preview changes, it says 'no changes take place'

**Comment by cheuschober** [comment]
@Tresanne,

Perhaps I'm confused. How are you developing your code? Are you using your local development environment or are you trying to use the editor in GitHub?

File: task_01.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+""" Just writing a doctring """
+
+BASE = raw_input('Which base color,"Seattle Gray" or "Manatee"? ')
+
+if BASE == "Seattle Gray":
+    ACCENT = raw_input('Which accent color,"Cermaic Glaze" or "Cumulus Nimbus" ' \
+                       '? ')
+
+    if ACCENT == "Cermaic Glaze":
+        HIGHLIGHT = raw_input('Which highlight color,"Basically White" ' \
```
Comment (cheuschober) [inline task_01.py]: You don't need the continuation `\` operator if you just group the strings, eg:

    HLIGHT = raw_input(('Some String'
                       'continued'))

**Comment by Tresanne** [comment]
@cheuschober I am using both. I basically finished most of the bulk work in the environment and am using the online editor to get rid of the syntax errors.

File: task_01.py (hunk 1 above)
Comment (Tresanne) [inline task_01.py]: ok Thanks. Will try it

**Comment by Tresanne** [comment]
@cheuschober How do I get rid of the trailing white space error in line 15 of task 1? I tried backspacing it but then the words i put on the second line moves and that is the only error that is preventing me from being finished.

**Comment by cheuschober** [comment]
@Tresanne, It means what it says -- you have whitespace (tabs, spaces or something like them) after the last line in that file. It looks like two spaces to me.

File: task_03.py [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+NAME = raw_input('What is your name? ')
+PRINCIPAL = raw_input('What is the principal of the loan?? ')
+PRINCIPAL = int(PRINCIPAL)
+TIME = raw_input('For how long is this being borrowed? ')
+TIME = int(TIME)
+QUALIFIED = raw_input('Are you pre-qualified for this loan? ')
+
+
+if PRINCIPAL <= 199999 and TIME <= 15 and QUALIFIED == 'yes':
```
Comment (cheuschober) [inline task_03.py]: @Tresanne Check out the video on how to structure these IF conditions together!

**Pull request merged**